packages/web-components/fast-foundation/src/accordion-item/accordion-item.ts

```diff
@@ -1,7 +1,33 @@
-import { attr, FASTElement, nullableNumberConverter } from "@microsoft/fast-element";
+import {
+    attr,
+    FASTElement,
+    html,
+    nullableNumberConverter,
+    observable,
+} from "@microsoft/fast-element";
 import { StartEnd } from "../patterns/start-end";
 import { applyMixins } from "../utilities/apply-mixins";
 
+const expandedIconTemplate = html`
+    <svg width="20" height="20" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg">
+        <path
+            fill-rule="evenodd"
+            clip-rule="evenodd"
+            d="M3.78 3h12.44c.43 0 .78.35.78.78v12.44c0 .43-.35.78-.78.78H3.78a.78.78 0 01-.78-.78V3.78c0-.43.35-.78.78-.78zm12.44-1H3.78C2.8 2 2 2.8 2 3.78v12.44C2 17.2 2.8 18 3.78 18h12.44c.98 0 1.78-.8 1.78-1.78V3.78C18 2.8 17.2 2 16.22 2zM14 9H6v2h8V9z"
+        />
+    </svg>
+`;
+
+const collapsedIconTemplate = html`
+    <svg width="20" height="20" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg">
+        <path
+            fill-rule="evenodd"
+            clip-rule="evenodd"
+            d="M16.22 3H3.78a.78.78 0 00-.78.78v12.44c0 .43.35.78.78.78h12.44c.43 0 .78-.35.78-.78V3.78a.78.78 0 00-.78-.78zM3.78 2h12.44C17.2 2 18 2.8 18 3.78v12.44c0 .98-.8 1.78-1.78 1.78H3.78C2.8 18 2 17.2 2 16.22V3.78C2 2.8 2.8 2 3.78 2zM11 9h3v2h-3v3H9v-3H6V9h3V6h2v3z"
+        />
+    </svg>
+`;
```

Comment:
@EisenbergEffect any idea what the performance implications are for making these observable by default vs. just using it as default slotted content? I'd prefer to not make this change without making a change across *every other* instance we have default slotted content as well. I know we discussed a potentially different implementation as well; would the other approach be favored above this?

Reply:
The new design system approach isn't mutually exclusive with this approach. It depends on how we want to shape the extensibility. I think we should delay creating the observable in light of the future configuration-based approach though. Additionally, I also think I forgot that the slot could always be targeted, which makes the use of observable for this particular scenario much less meaningful.

@khamudom Apologies for the distraction 😢

Reply:
@EisenbergEffect made an update based on our discussion and your suggestions.